automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -69,7 +69,6 @@ final class LeonardoSuite extends Suites(
   new NotebookDataSyncingSpec,
   new NotebookGATKSpec,
   new NotebookHailSpec,
-  new NotebookLocalizeFileSpec,
```

Comment:
Simply removing this test as it's no longer relevant. It failed in https://broadworkbench.atlassian.net/browse/IA-1380

Reply:
I added this back - the old localize is still used for legacy clusters.